### PR TITLE
CI pt. 2: Fixes

### PR DESCRIPTION
**Pull request by kedhammar**
To be merged after https://github.com/NationalGenomicsInfrastructure/genomics-status/pull/882

### CODE REVIEW

File: status/projects.py [hunk 1]
```diff
@@ -1033,26 +1028,19 @@ def get(self, projectid):
                     self.application.settings["charon"]["api_token"]
                 )
             }
-            project_url = "{}/api/v1/project/{}".format(
+            "{}/api/v1/project/{}".format(
                 self.application.settings["charon"]["url"], projectid
             )
```
Comment (aanil) [inline status/projects.py:1033]:
```suggestion
```
This is not used anywhere

File: status/authorization.py [hunk 2]
```diff
@@ -34,7 +35,7 @@ def get(self):
                 if url is None:
                     url = "/"
             else:
-                url = "/unauthorized?email={0}&contact={1}".format(
+                url = "/unauthorized?email={}&contact={}".format(
```
Comment (aanil) [inline status/authorization.py:38]: Why not replace this with an f-string as well?

**Comment by aanil** [comment]
Should the ruff-format check be failing?

**Comment by kedhammar** [comment]
> Should the ruff-format check be failing? Or is it that it will be fixed in #884 ?

This is `ruff check` which looks for smelly code, ´ruff format´ is the next PR :)

File: status/projects.py (hunk 1 above)
Comment (kedhammar) [reply status/projects.py:1033]: It's just a string chilling there huh 😆

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**